starboard/common/file.h

```diff
@@ -22,7 +22,19 @@
 
 #include "starboard/file.h"
 
+#include <fcntl.h>
+#include <sys/stat.h>
 #include <sys/types.h>
+#include <unistd.h>
+
+#include "starboard/common/log.h"
+
+#ifdef _WIN32
```

Comment:
why do we need this?

Reply:
This is to resolve the issue we saw when cobalt/base/localized_strings.cc failed to access sb_open for Windows, we should further investigate and clean up these macros for Windows.